src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -1857,13 +1834,14 @@ let internal_commands logger =
 
 let mina_commands logger ~itn_features =
   [ ("accounts", Client.accounts)
-  ; ("daemon", daemon logger)
+  ; ("daemon", daemon ~itn_features logger)
   ; ("client", Client.client)
   ; ("advanced", Client.advanced ~itn_features)
   ; ("ledger", Client.ledger)
   ; ("libp2p", Client.libp2p)
   ; ( "internal"
-    , Command.group ~summary:"Internal commands" (internal_commands logger) )
+    , Command.group ~summary:"Internal commands"
```

Comment:
Commit cb395b201406816633a6f434d9c27b8d11ca9c02 also propagates `~itn_features` down `daemon` and `internal_commands -> replay_blocks` so that it's read from compile time config in the only place in `mina_cli_entrypoint`